test/anacondalib.py

```diff
@@ -50,6 +56,17 @@ def setUp(self):
 
         super().setUp()
 
+        # Add installation target disk
+        backing_file = None if not self.disk_image else os.path.join(BOTS_DIR, f"./images/{self.disk_image}")
+        self.add_disk(15, backing_file)
+        # Select the disk as boot device
+        self._execute(f"virt-xml -c qemu:///session {self.machine.label} --edit --boot hd")
+
+        m = self.machine
+        b = self.browser
+        s = Storage(b, m)
+        s.dbus_scan_devices()
```

Comment:
You can write this in one line without the variables, but the more interesting question is: Is anaconda already running at this point? I suppose yes, otherwise this shouldn't be necessary.

Reply:
Anaconda is running and the Web UI is ready to be interacted with. 
This is waited for in the VM spawning script https://github.com/rhinstaller/anaconda-webui/blob/main/test/machine_install.py#L170